examples/http/client.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+require 'rubygems'
+require 'bundler/setup'
+require 'faraday'
+# Require otel-ruby
+require 'opentelemetry/sdk'
+
+# Allow setting the host from the ENV
+host = ENV.fetch('HTTP_EXAMPLE_HOST', '0.0.0.0')
+
+SDK = OpenTelemetry::SDK
+OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+
+# Configure tracer
+exporter = SDK::Trace::Export::ConsoleSpanExporter.new
+processor = SDK::Trace::Export::SimpleSpanProcessor.new(exporter)
+OpenTelemetry.tracer_factory.add_span_processor(processor)
+tracer = OpenTelemetry.tracer_factory.tracer('faraday', 'semver:1.0')
+formatter = OpenTelemetry.tracer_factory.http_text_format
+
+connection = Faraday.new("http://#{host}:4567")
+url = '/hello'
+
+# For attribute naming, see:
+# https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md#http-client
+
+# Span name should be set to URI path value:
+tracer.in_span(url, kind: :client) do |span|
+  span.set_attribute('component', 'http')
+  span.set_attribute('http.method', 'GET')
+  span.add_event(name: 'request http.client.get')
+
+  response = connection.get(url) do |request|
+    # Inject context into request headers
+    formatter.inject(span.context, request.headers) do |carrier, key, value|
```

Comment:
You can omit the block now that #140 has merged.

Reply:
@mwear yep, thank you, it's done!